backend/app/assets/stylesheets/spree/backend/components/_navigation.scss

```diff
@@ -1,169 +1,102 @@
-// Navigation
-//---------------------------------------------------
-.inline-menu {
```

Comment:
With removing the `.inline-menu` styles you have default margin and padding of `ul` back in, which causes the `#content-header` to change height, dependent from presence of an inline menu.

![solidus administration users 2015-11-18 15-36-50](https://cloud.githubusercontent.com/assets/42868/11243777/34cf9a50-8e0a-11e5-8dc2-f384a9c5247d.png)